### DISCUSSION TITLE
Label Names and Author Names

### DISCUSSION THREAD

**Discussion by petsi97** [snap-stanford/ogb#259] [MAG240M-LSC]
Hello everyone.

I downloaded and read of the provided data (mag240m) and I saw that there is no extra information about the Label and Author Names. I understand that only the IDs are presented for simplicity, in order to manage the data more easily. Despite that, I was wondering if  the real Νames of the Subject Areas and Authors  are available (ID -> Name), for having a better represantation and for evaluating our results in case studies.

Thank you in advance.

**Comment by weihua916** [comment]
We are not providing the names/texts in order to keep the test label integrity, i.e., test labels cannot be easily revealed by manually looking into the arXiv page. Hence, we will need to anonymize our nodes as much as possible.
